spec/myst/env_spec.mt

```diff
@@ -0,0 +1,117 @@
+require "stdlib/spec.mt"
+
+initial_env = ENV.to_map
+
+# Keys and values used for testing
+SHAKE_KEY = "Shakespeare"
+SHAKE     = "To be, or not to be: that is the question"
+EIN_STEIN_KEY = "Einstein"
+EIN_STEIN     = "two things are infinite: the universe and human stupidity"
+
+TEST_ENV = { 
+  <SHAKE_KEY>:     SHAKE, 
+  <EIN_STEIN_KEY>: EIN_STEIN, 
+  <"HOME">: "/Users/Bob",
+  <"PWD">:  "/Users/Bob/Projects/myst-lang/myst"
+}
+
+def set_test_env
+  ENV.set(TEST_ENV)
+end
+
+def random_key
+  key = ""
+  16.times {key += Random.rand(10).to_s}
+  key
+end
+
+describe("ENV[]") do
```

Comment:
Just a nitpick. Can all of the `ENV` specs get put under a general `describe("ENV") do`, and then have separate, nested `describe` blocks for each method?

Reply:
Oh I'd love to, but I thought the current spec module didn't support that yet?

Reply:
you can nest them, it just doesn't format the name super nicely in the error messages.

Reply:
Right that was what it was